app/models/DisplayModels.scala

```diff
@@ -53,9 +53,52 @@ case class ExistingProtectionDisplayModel(
 
 case class ExistingProtectionsDisplayModel(
     activeProtection: Option[ExistingProtectionDisplayModel],
-    otherProtections: Seq[ExistingProtectionDisplayModel]
+    inactiveProtections: ExistingInactiveProtectionsDisplayModel
 )
 
+case class ExistingInactiveProtectionsDisplayModel(
+    dormantProtections: ExistingInactiveProtectionsByType,
+    withdrawnProtections: ExistingInactiveProtectionsByType,
+    unsuccessfulProtections: ExistingInactiveProtectionsByType,
+    rejectedProtections: ExistingInactiveProtectionsByType,
+    expiredProtections: ExistingInactiveProtectionsByType
+) {
+
+  def isEmpty: Boolean =
+    dormantProtections.isEmpty && withdrawnProtections.isEmpty && unsuccessfulProtections.isEmpty && rejectedProtections.isEmpty && expiredProtections.isEmpty
+
+  def nonEmpty: Boolean = !isEmpty
+
+}
+
+object ExistingInactiveProtectionsDisplayModel {
+
+  def empty =
+    ExistingInactiveProtectionsDisplayModel(
+      ExistingInactiveProtectionsByType.empty,
+      ExistingInactiveProtectionsByType.empty,
+      ExistingInactiveProtectionsByType.empty,
+      ExistingInactiveProtectionsByType.empty,
+      ExistingInactiveProtectionsByType.empty
+    )
+
+}
+
+case class ExistingInactiveProtectionsByType(
+    protections: (String, Seq[ExistingProtectionDisplayModel])*
```

Comment:
What do you think about changing this varargs into a `Seq`? I found varargs tricky to work with in some instances. If you need a constructor accepting varargs, you can always create an `apply` method in the companion object.